pkg/virtctl/sev/attestation_test.go

```diff
@@ -0,0 +1,289 @@
+package sev_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/api"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/pkg/virtctl/sev"
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("SEV Attestation", func() {
+	const vmiName = "testvm"
+	var vm *v1.VirtualMachine
+	var vmi *v1.VirtualMachineInstance
+	var vmInterface *kubecli.MockVirtualMachineInterface
+	var vmiInterface *kubecli.MockVirtualMachineInstanceInterface
+	var ctrl *gomock.Controller
+	var workDir string
+	var err error
+
+	BeforeEach(func() {
+		vm = kubecli.NewMinimalVM(vmiName)
+		vmi = api.NewMinimalVMI(vmiName)
+		ctrl = gomock.NewController(GinkgoT())
+		workDir, err = ioutil.TempDir("", "sev-test")
```

Comment:
You can use `GinkgoT().TempDir()` here and you will not need to do the cleanup in `AfterEach`

Reply:
Not sure why this was resolved but I'd also suggest to used the TempDIr creation from ginkgo or any particular reason for not using it